packages/analytics-js/src/app/RudderAnalytics.ts

```
@@ -50,55 +51,62 @@
   // Singleton with constructor bind methods
   constructor() {
-    if (RudderAnalytics.globalSingleton) {
-      // START-NO-SONAR-SCAN
-      // eslint-disable-next-line no-constructor-return
-      return RudderAnalytics.globalSingleton;
-      // END-NO-SONAR-SCAN
+    try {
+      if (RudderAnalytics.globalSingleton) {
+        // START-NO-SONAR-SCAN
+        // eslint-disable-next-line no-constructor-return
+        return RudderAnalytics.globalSingleton;
```

Biome

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)